static/js/plotjs/stackSpectra.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable */
```

Comment:
Is this to bypass the import error or something else?

Reply:
Hmm might be for undeclared variable? Since `bin_slider` is passed in by bokeh and isn't defined in that file. I'm not actually sure - that line is left over from copy-pasting the logic for the photometry binning :)

Reply:
Better to disable individual lines as needed.

Reply:
Can we reuse binning from photometry, or are they too different?

Reply:
The binning from photometry is fairly different. Namely, the photometry binning must deal with both the photometry points and the associated error bars when replotting based on binning changes, whereas the spectroscopy only cares about the one line. Also, the photometry is a scatter plot where the spectroscopy is a step function so that also slightly alters the data access methodology. I think it would be simpler to just keep them separate.